src/main/java/seedu/address/model/task/TaskName.java

```diff
@@ -7,7 +7,11 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidTaskName(String)}
  */
 public class TaskName {
+<<<<<<< HEAD
```

Comment:
Merge conflict here has not been resolved. Should be resolved as soon as possible.

Reply:
Okay, it is now resolved.